content/docs/conditional-rendering.md

```diff
@@ -44,13 +43,13 @@ ReactDOM.render(
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+यह उदहारण `isLoggedIn` प्रोप के वैल्यू के अनुसार अलग अलग ग्रीटिंग्स रेंडर करता है।
 
-### Element Variables {#element-variables}
+### एलिमेंट वेरिएबल्स {#element-variables}
 
-You can use variables to store elements. This can help you conditionally render a part of the component while the rest of the output doesn't change.
+आप एलिमेंट्स को स्टोर करने के लिए वेरिएबल्स का उसे कर सकते हो। जिससे आपको कंडीशनल रेंडरिंग में मदद होगी और बाकि का आउटपुट चेंज नहीं होगा। 
 
-Consider these two new components representing Logout and Login buttons:
+अब लॉगिन और लॉगआउट बटन्स नए कौम्पोनॅन्टस को देखो :
```

Comment:
not the right translation

Reply:
लॉगिन और लॉगआउट बटन्स के नए कौम्पोनॅन्टस पर विचार करें

Reply:
done

Reply:
लॉगआउट और लॉगिन बटन्स का प्रतिनिधित्व करने वाले इन दो नए कौम्पोनॅन्टस पर विचार करें: